### ISSUE TITLE
Deno has panicked: I had problem running deno with docker in production on CentOS 7

### ISSUE THREAD

**Issue by lucas-bogos** [denoland/deno#19701]
```
============================================================
Platform: linux x86_64
Version: 1.34.3
Args: ["deno", "run", "--allow-net", "--allow-read", "--allow-env", "src/main.ts"]
thread 'main' panicked at 'OS can't spawn worker thread: Operation not permitted (os error 1)', /home/runner/work/deno/deno/core/task.rs:71:16
stack backtrace:
   0: rust_begin_unwind
             at /rustc/9eb3afe9ebe9c7d2b84b71002d44f4a0edac95e0/library/std/src/panicking.rs:575:5
   1: core::panicking::panic_fmt
             at /rustc/9eb3afe9ebe9c7d2b84b71002d44f4a0edac95e0/library/core/src/panicking.rs:64:14
   2: once_cell::imp::OnceCell<T>::initialize::{{closure}}
   3: once_cell::imp::initialize_or_wait
   4: deno::cache::caches::Caches::make_db
   5: deno::cache::caches::Caches::dep_analysis_db
   6: deno::factory::CliFactory::caches
   7: deno::factory::CliFactory::parsed_source_cache
   8: deno::factory::CliFactory::emitter
   9: deno::factory::CliFactory::create_cli_main_worker_factory::{{closure}}
  10: deno::tools::run::run_script::{{closure}}
  11: <deno_core::task::MaskFutureAsSend<F> as core::future::future::Future>::poll
  12: tokio::runtime::task::raw::poll
  13: deno::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

**Comment by dsherret** [comment]
This seems like some operating system restriction where Deno can't spawn new threads. Is it possible to remove this restriction in the docker container?

**Comment by FinlayDaG33k** [comment]
Had the same issue under Docker 20.10.8 (Ubuntu 20.04) as well, updating your Docker runtime should fix it (I'm at 24.0.2 right now and it works fine).

If memory serves right, there was some kind of issue with the default seccomp profile not working with a certain version of glibc.

**Comment by lucas-bogos** [comment]
I did the docker update and it worked fine. Thanks!
